kiwi/bootloader/config/grub2.py

```diff
@@ -44,7 +44,7 @@ class BootLoaderConfigGrub2(BootLoaderConfigBase):
     def post_init(self, custom_args):
         self.custom_args = custom_args
         arch = platform.machine()
-        if arch == 'x86_64':
+        if arch == 'x86_64' or arch.startswith('ppc64'):
```

Comment:
we how now ppc64 and ppc64le former is big endian latter is little endian, booting wise those are compatible